app/src/main/java/ch/epfl/sdp/item/Market.java

```diff
@@ -0,0 +1,62 @@
+package ch.epfl.sdp.item;
+
+import android.util.Pair;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Random;
+
+import ch.epfl.sdp.entity.EntityType;
+import ch.epfl.sdp.entity.Player;
+import ch.epfl.sdp.logic.RandomGenerator;
+import ch.epfl.sdp.map.Displayable;
+import ch.epfl.sdp.map.GeoPoint;
+import ch.epfl.sdp.map.MapsActivity;
+

```

Comment:
add javadoc for the classes and for the public methods

Reply:
I'll add Javadoc for the new public methods